fs/mount/fs_procfs_mount.c

```diff
@@ -248,6 +248,10 @@ static int blocks_entry(FAR const char *mountpoint,
 
   if (!info->header)
     {
+#if defined(CONFIG_HAVE_LONG_LONG) && !defined(CONFIG_LIBC_LONG_LONG)
```

Comment:
since the code increase of long long is very little, look like it's better to remove CONFIG_LIBC_LONG_LONG and use CONFIG_HAVE_LONG_LONG instead.